standalone/content/js/standaloneRoomViews.jsx

```diff
@@ -722,7 +735,7 @@ loop.standaloneRoomViews = (function(mozL10n) {
     render: function() {
       var displayScreenShare = !!(this.state.receivingScreenShare ||
         this.props.screenSharePosterUrl);
-
+      var overlay = (!this.state.introSeen) ? <IntroOverlayView joinRoom={this.joinRoom} /> : false;
```

Comment:
s/false/null to match prevailing style.  Also, please invert the test for readability.